og.module

```diff
@@ -303,6 +303,38 @@ function og_entity_type_alter(array &$entity_types) {
   }
 }
 
+/**
+ * Implements hook_theme().
+ */
+function og_theme($existing, $type, $theme, $path) {
+  return [
+    'og_member_count' => [
+      'variables' => [
+        'count' => 0,
+        'membership_states' => [],
+        'group' => NULL,
+      ],
+    ],
+  ];
+}
+
+/**
+ * Prepares variables for member count templates.
+ *
+ * Default template: og-member-count.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - count: An integer representing the number of members in the group.
+ *   - group: The group, which is a content entity.
+ */
+function template_preprocess_og_member_count(array &$variables) {
```

Comment:
I don't see the reason of this preprocess. We can pass the label directly from the block plugin.

Reply:
This is intended to demonstrate how and end user would change this block into something useful for them.